tx-pool/src/component/entry.rs

```diff
@@ -90,7 +91,7 @@ impl TxEntry {
     }
 
     /// Returns a sorted_key
-    pub fn as_sorted_key(&self) -> AncestorsScoreSortKey {
+    pub fn as_score_key(&self) -> AncestorsScoreSortKey {
```

Comment:
Do we need to change this method's comment?

Reply:
I think `as_score_key` maybe a better name, since we may have other sort keys.